src/main/kotlin/org/rust/cargo/commands/ProjectDescription.kt

```diff
@@ -0,0 +1,79 @@
+package org.rust.cargo.commands
+
+import com.intellij.util.PathUtil
+import org.rust.cargo.commands.impl.Package
+import org.rust.cargo.commands.impl.Project
+import java.io.File
+import java.util.*
+
+class ProjectDescription(private val project: Project) {
```

Comment:
It's better call it `CargoProjectDescription` to clearly attach it to the topic.